test/parallel/test-node-run.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const common = require('../common');
+const { it, describe } = require('node:test');
+const assert = require('node:assert');
+
+const fixtures = require('../common/fixtures');
+const isLinuxOrOSX = common.isLinux || common.isOSX;
+
+describe('node run [command]', () => {
+  it('returns error on non-existent command', async () => {
+    const child = await common.spawnPromisified(
+      process.execPath,
+      [ 'run', 'test'],
+      { cwd: __dirname },
+    );
+    assert.match(child.stdout, /Can't read package\.json/);
+    assert.strictEqual(child.stderr, '');
+    assert.strictEqual(child.code, 1);
+  });
+
+  it('runs a valid command', { skip: !isLinuxOrOSX }, async () => {
```

Comment:
So we have no tests for this feature in Windows? Does it even work in Windows? I don’t think we want to add it without Windows support.

Reply:
@anonrig If you enable the test under Windows, I will volunteer for the pain of windows debugging, if needed.

Reply:
The tests run a `/bin/sh` script and I have no idea how to run a similar script in Windows. I can remove this `skip` but I leave myself to the hands of @lemire.

Reply:
Done. We have tests running under Windows.